lib/memory_analyzer/heap_analyzer/parser.rb

```diff
@@ -93,6 +95,15 @@ def string_pool
       def string_pool_intern(string)
         string_pool.add(string)[string]
       end
+
+      def io_klass
```

Comment:
might as well call it `io_class`

Reply:
psh... all the cool kids spell klass with a K...

kidding, fair enough